src/services/redeem_and_unstake/ProveCoGatewayService.ts

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable import/no-unresolved */
+
+import assert from 'assert';
+import BigNumber from 'bignumber.js';
+import Web3 from 'web3';
+
+import { interacts } from '@openst/mosaic-contracts';
+import { ProofGenerator } from '@openst/mosaic-proof';
+
+import { ORIGIN_GAS_PRICE} from '../../Constants';
+import Logger from '../../Logger';
+import AuxiliaryChain from '../../models/AuxiliaryChain';
+import Observer from '../../observer/Observer';
+import GatewayRepository from '../../repositories/GatewayRepository';
+import {MessageDirection, MessageRepository} from '../../repositories/MessageRepository';
```

Comment:
Also fix other linter issues.
```suggestion
import { MessageDirection, MessageRepository } from '../../repositories/MessageRepository';
```

Reply:
Done!